specification/hugr.md

```diff
@@ -1075,8 +1079,8 @@ equality constraint of `typeof(b) ~ Bool`.
 ### Types of built-ins
 
 We will provide some built in modules to provide basic functionality.
-I’m going to define them in terms of extensions. We have the “builtin”
-extension which should always be available when writing hugr plugins.
+I’m going to define them in terms of resources. We have the “builtin”
+resource which should always be available when writing hugr plugins.
```

Comment:
is this now the only mention of resources? does it need defining elsewhere?

Reply:
Indeed, I've changed this to "We will define them in terms of extensions"